docs/SetupAzureDatabase.md

```diff
@@ -51,8 +51,126 @@ Autoscale provisioned throughput in Azure Cosmos DB allows you to scale the thro
 The throughput is scaled based on the usage, without impacting the availability, latency, throughput, or performance of the workload.
 By default, the scalardl schema tool enables autoscale of RU for all tables: RU is scaled in or out between 10% and 100% of a specified RU depending on a workload.
 
+### Backup and Restore
+
+Azure Cosmos DB automatically takes backups of your data at regular intervals. The automatic backups are taken without affecting the performance or availability of the database operations. All the backups are stored separately in a storage service. 
```

Comment:
As I said, the automatic backup cannot be used for Scalar DB since it breaks the transaction consistency of data.
We need to do pausing (or stopping) an application and PITR.

Reply:
Ok, I will update the document.